packages/ui/components/data-table/DataTableFilter.tsx

```diff
@@ -23,7 +22,8 @@ interface DataTableFilter<TData, TValue> {
   options: {
     label: string;
     value: string;
-    icon?: LucideIcon;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    icon?: any;
```

Comment:
do we have a type here?

Reply:
Yeah. This should be either a ReactNode or IconName